dataset/src/main/scala/frameless/ops/AggregateTypes.scala

```diff
@@ -0,0 +1,19 @@
+package frameless
+package ops
+
+import shapeless._
+
+trait AggregateTypes[V, U <: HList] {
```

Comment:
Cool :). In the commit I liked above I make it work with a type alias:

    trait Foo[A, B]

    type FooString[B] = Foo[String, B]

    implicitly[
      Comapped.Aux[
        FooString[Int] :: HNil,
        FooString,
        Int :: HNil
      ]
    ]